src/surabhi/coding_exam_1/IntegerNumArray.java

```diff
@@ -0,0 +1,41 @@
+package surabhi.coding_exam_1;
+
+/*Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to the target.
+
+Input: nums = [2,11,15,7,4], target = 9
+
+Output: [0,3]*/
+
+public class IntegerNumArray {
+	int targetNum=9;
+	int count=0;
+	
+	int[] getSum(int[] num) {
+		int [] arr=new int[num.length];
+		for(int index=0; index<num.length;index++) {
+			for(int innerIndex=0; innerIndex<num.length;innerIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files